### PR TITLE
Add tenant, account_name, funds to create_account

### PR DESCRIPTION
**Pull request by elv-wayne**
Creates a new account with funds, account_name and tenantId.

./elv-live create_account 0.1 test iten3gVtWToi2hmCKfwxXfxtkY46HFDv
Create Account

Network: demov3
mnemonic: >-
  XXX XXX XXX
privateKey: '0xXXXXXXX'
address: '0xXXXX'
accountName: test
balance: '0.058687656'

### Also Added account_show
./elv-live account_show
Account Show

Network: demov3
tenantId: iten3gVtWToi2hmCKfwxXfxtkY46HFDv
walletAddress: '0xa1d85Bb5bD07aDe0db73c2c013D3611DB69c40BC'
userWalletObject:
  libraryId: ilib3ANoVSzNA3P6t7abLR69ho5YPPZU
  objectId: iq__3Fn4en4mRXRiPRzuMivgdRK1iweT
userMetadata:
  commit:
    author: '0x211446b91f03a2657582073ba8a1a895ace2c187'
    author_address: '0x211446b91f03a2657582073ba8a1a895ace2c187'
    message: Replace user metadata
    timestamp: '2022-04-11T20:34:49.194Z'
  public:
    name: test

### CODE REVIEW

File: src/EluvioLive.js [hunk 1]
```diff
@@ -43,8 +43,12 @@ class EluvioLive {
     this.client.ToggleLogging(false);
   }
 
-  async InitNew() {
-    this.client = await ElvClient.FromConfigurationUrl({
+  async CreateAccount({ funds = 0.1, accountName, tenantId }) {
```
Comment (elv-kevin) [inline src/EluvioLive.js:46]: Client and core consider < 0.1 eth to be effectively 'out of funds'. Recommend bumping this to at least 0.25, if not 1.

Comment (elv-wayne) [reply src/EluvioLive.js]: Bumped to .25, thanks.

**Review by elv-serban — APPROVED** [review]
Looks good!

**Review by elv-serban — APPROVED** [review]
Looks good!

**Comment by elv-serban** [comment]
The current convention actually take the 'tenant admins ID' not the actual 'tenant ID'
Let's rename the option:
```
   tenant_admins   Tenant Admins group ID (as itenXXX)
```

File: src/EluvioLive.js [hunk 2]
```diff
@@ -43,8 +43,22 @@ class EluvioLive {
...
+   * Creates a new account including wallet object and contract.
+   * Current client must be initialized and funded.
+   *
+   * @namedParams
+   * @param {number} funds - The amount in ETH to fund the new account.
+   * @cauth {string} accountName - The name of the account to set in it's wallet metadata (Optional)
+   * @cauth {string} tenantId - The tenant ID to set for the user's wallet (Optional)
+   * @return {Promise<Object>} - An object containing the new account mnemonic, privateKey, address, accountName, balance
+   */
+  async CreateAccount({ funds = 0.25, accountName, tenantId }) {
```
[the remark this answers is not in the document]
Comment (elv-wayne) [reply src/EluvioLive.js]: I was going to leave it since we want to move it but it's better to be totally consistent. Done.

**Comment by elv-wayne** [comment]
> 

Renamed to tenant_admins

**Review by elv-serban — APPROVED** [review]
Looks good!

**Pull request merged**